### DISCUSSION TITLE
Bootable image

### DISCUSSION THREAD

**Discussion by erkinalp** [SakanaAI/AI-Scientist#103] [Ideas]
Not kidding, have a bootable ISO image that automatically boots up from the AI scientist and is appropriately configured to store temporary and final results in a pre-configured location and automatically uploads the paper.

**Comment by BradKML** [comment]
You mean Docker, or genuinely a physical CD? If you need to customize, would be tough to do Docker tho https://github.com/SakanaAI/AI-Scientist/discussions/109

**Reply by erkinalp** [reply]
An ISO image, to network boot a whole cluster of AI Scientist agents.

**Reply by BradKML** [reply]
But AI Scientist does not contain an OS (or need it), so why ISO? You can set the cluster up with a bootable USB and have the code of the AI Scientists on another USB. OR you can partition the USB into a bootable partition, flash that part as normal, and another research/data partition that is untouched until you do experiments.
If we are talking network boot, then the same rules apply, just boot a regular OS and then mass execute a shell script that install AI Scientist across multiple machines. Applications should be separate from OS and hardware.
